pkg/ignition/ignition.go

```diff
@@ -132,3 +166,178 @@ func parseKubeletArgs(unitContents string) (map[string]string, error) {
 	}
 	return kubeletArgs, nil
 }
+
+// GetBootstrapFiles returns the paths of all the files needed to bootstrap kubelet
+func (ign *Ignition) GetBootstrapFiles() []string {
```

Comment:
I dont see this function being used in this commit, please hold off on adding it until it is used

Reply:
Removed it altogether